src/frontend/src/binder/expr/function.rs

```diff
@@ -290,7 +290,13 @@ impl Binder {
             // TODO: choose which pg version we should return.
             "version" => return Ok(ExprImpl::literal_varchar("PostgreSQL 13.9-RW".to_string())),
             // non-deterministic
-            "now" => ExprType::Now,
+            "now" => {
```

Comment:
Perhaps one solution is to only bind the session_timestamp if the session is batch?

Reply:
> Perhaps one solution is to only bind the session_timestamp if the session is batch?

how?

Reply:
> Perhaps one solution is to only bind the session_timestamp if the session is batch?
> 
> ~session~ -> query

fixed